stk/calculators/energy/energy_calculators.py

```diff
@@ -377,3 +382,887 @@ def energy(self, mol, conformer=-1):
+
+        electronic_temperature : :class:`int`, optional
+            Electronic temperature to use (in K).
+
+        solvent : :class:`str`, optional
+            Solvent to use in GBSA implicit solvation method.
+            For details see
+            https://xtb-docs.readthedocs.io/en/latest/gbsa.html
+
+        solvent_grid : :class:`str`, optional
+            Grid level to use in SASA calculations for GBSA implicit
+            solvent.
+            Can be one of ``'normal'``, ``'tight'``, ``'verytight'``
+            or ``'extreme'``
+            For details see
+            https://xtb-docs.readthedocs.io/en/latest/gbsa.html
+
+        charge : :class:`int`, optional
+            Formal molecular charge.
+
+        unpaired_electrons : :class:`int`, optional
+            Number of unpaired electrons.
+
+        use_cache : :class:`bool`, optional
+            If ``True`` :meth:`energy` will not run twice on the same
+            molecule and conformer.
+
+        unlimited_memory : :class: `bool`, optional
+            If ``True`` :meth:`energy` will be run without constraints
+            on the stacksize. If memory issues are encountered, this
+            should be ``True``, however this may raise issues on
+            clusters.
+
+        """
+        self.xtb_path = xtb_path
+        self.gfn_version = gfn_version
+        self.output_dir = output_dir
+        self.num_cores = str(num_cores)
+        self.electronic_temperature = str(electronic_temperature)
+        self.solvent = solvent
+        if self.solvent is not None:
+            self.solvent = solvent.lower()
+            valid_xtb_solvent(
+                gfn_version=self.gfn_version,
+                solvent=self.solvent
+            )
+        self.solvent_grid = solvent_grid
+        self.charge = str(charge)
+        self.unpaired_electrons = str(unpaired_electrons)
+        self.unlimited_memory = unlimited_memory
+
+        self.total_energies = {}
+        self.homo_lumo_gaps = {}
+        self.fermi_levels = {}
+        self.homo_lumo_orbitals = {}
+        self.qonly_dipole_moms = {}
+        self.full_dipole_moms = {}
+        self.qonly_quadrupole_moms = {}
+        self.qdip_quadrupole_moms = {}
+        self.full_quadrupole_moms = {}
+        super().__init__(use_cache=use_cache)
+
+    def _get_properties(self, mol, conformer, output_file):
+        """
+        Extracts properties from GFN-xTB energy calculation.
+
+        Parameters
+        ----------
+        mol : :class:`.Molecule`
+            The :class:`.Molecule` whose energy is to be calculated.
+
+        conformer : :class:`int`, optinal
+            The conformer of `mol` to use.
+
+        output_file : :class: `str`
+            Name of output file with xTB results.
+
+        Returns
+        -------
+        None : :class:`NoneType`
+
+        """
+        xtbext = XTBExtrators(output_file=output_file)
+
+        # Get properties from output file.
```

Comment:
from output_file (ie you are referring to the variable directly)
or 
from the output file (ie you are referring to the concept or whatever)